scripts/deploy-blobstreamx.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/bash
+
+# Load env variable from `.env` only if they're not already set
+if [ -z "$STARKNET_KEYSTORE" ] || [ -z "$STARKNET_ACCOUNT" ]; then
+  source ../.env
+fi
+
+# Check if required env variables are set, if not exit
+if [ -z "$STARKNET_KEYSTORE" ]; then
+  echo "Error: STARKNET_KEYSTORE is not set."
+  exit 1
+elif [ -z "$STARKNET_ACCOUNT" ]; then
+  echo "Error: STARKNET_ACCOUNT is not set."
+  exit 1
+fi
+
+# Constants
+SIERRA_FILE=../target/dev/blobstream_sn_blobstreamx.contract_class.json
```

Comment:
I usually run the scripts from the top level dir. Can we make this less path dependent? Maybe `git rev-parse --show-toplevel`

Reply:
Made the change in #137